frontend/views/containers/dashboard/GroupMembersAllModal.vue

```diff
@@ -5,48 +5,33 @@ modal-base-template.has-background(ref='modal' :fullscreen='true')
       i18n.is-title-2.c-title(tag='h2') Group members
 
     .card.c-card
-      form(
-        @submit.prevent='submit'
+      search(
+        v-if='needsIncome || activeTab !== "todo"'
+        :placeholder='L("Search payments...")'
+        :label='L("Search for a payment")'
+        v-model='searchText'
       )
-        label.field
-          i18n.sr-only Search for a member
-          .input-combo
-            .is-icon(:aria-label='L("Search")')
-              i.icon-search
-            input.input(
-              type='text'
-              name='search'
-              data-test='search'
-              placeholder='Search...'
-              v-model='form.search'
-            )
-            button.is-icon-small(
-              v-if='form.search'
-              aria-label='Clear search'
-              @click='form.search = null'
-            )
-              i.icon-times
 
-        i18n.c-member-count.has-text-1(
-          v-if='form.search && searchCount > 0'
-          tag='div'
-          :args='{ searchCount: `<strong>${searchCount}</strong>`, result: `<strong>${searchCount === 1 ? L("result") : L("results")}</strong>`, searchTerm: `<strong>${form.search}</strong>`}'
-          data-test='memberSearchCount'
-          compile
-        ) Showing {searchCount} {result} for "{searchTerm}"
-
-        i18n.c-member-count.has-text-1(
-          v-if='form.search && searchCount === 0'
-          tag='div'
-          :args='{searchTerm: `<strong>${form.search}</strong>`}'
-        ) Sorry, we couldn't find anyone called "{searchTerm}"
-
-        i18n.c-member-count.has-text-1(
-          v-if='!form.search'
-          tag='div'
-          :args='{ groupMembersCount }'
-          data-test='memberCount'
-        ) {groupMembersCount} members
+      i18n.c-member-count.has-text-1(
+        v-if='searchText && searchCount > 0'
+        tag='div'
+        :args='{ searchCount: `<strong>${searchCount}</strong>`, result: `<strong>${searchCount === 1 ? L("result") : L("results")}</strong>`, searchTerm: `<strong>${searchText}</strong>`}'
+        data-test='memberSearchCount'
+        compile
+      ) Showing {searchCount} {result} for "{searchTerm}"
```

Comment:
This string has the same issue as the other one, where variable is being used to insert an individually translated word. In general, this should be avoided, because we cannot assume to know how other languages will pluralize works.

For example, in English, if the `searchCount === 1`, we say "1 result", but another language might have the condition backwards, where it says effectively "1 results", or some other weirdness. Similarly, to the translator, the variables are assumed to be values, not words, so they might have difficulty figuring out what to do with `{result}`, thinking it represents a number or some other value.

In these situations it's again best to build full sentences:

- `Showing 1 result for "{searchTerm}"`
- `Showing {searchCount} results for "{searchTerm}"`

This gives the translators all the information they need to figure out what sort of sentence this is and how it should be translated natively. Notice we do not use `{searchCount}` for the first case.

So here I recommend not using `i18n`, and instead using the `L` function inside of a computed property, to return one of the two strings in the unordered list above ☝️ 

As a general rule, I would look at the use of `L()` functions inside of the `:args` of `i18n` tags as a sign that whole-sentence translation through computed properties should be used for this sentence instead.